cmd/src/extensions_list.go

```diff
@@ -63,6 +63,7 @@ Examples:
       nodes {
         ...RegistryExtensionFields
       }
+			error
```

Comment:
the indentation here is a bit off

Reply:
![Screenshot 2022-08-17 at 13 34 05](https://user-images.githubusercontent.com/458591/185108555-9a7bec42-92ef-4455-918d-675a13ea7aee.png)

ah 🤦